test/accessibility.html

```diff
@@ -70,6 +70,10 @@
           expect(vaadinCheckbox.getAttribute('aria-checked')).to.be.eql('mixed');
         });
 
+        it('should set aria-checked to false when setting checked property is null', () => {
```

Comment:
"when setting checked property is null" sounds wrong. Replacing "is" with "to" would be ok, but here's a suggestion making it slightly more consistent with the other descriptions above too.
```suggestion
        it('should set aria-checked to "false" when checked is set to null', () => {
```

Reply:
thank you for correcting it. I changed it.